schema/me/update_collector_profile.js

```diff
@@ -16,20 +15,15 @@ export default mutationWithClientMutationId({
     self_reported_purchases: {
       type: GraphQLString,
     },
+    intents: {
+      type: new GraphQLList(GraphQLString),
+    },
```

Comment:
Is this actually a freeform list of values? If not then this should use an enum instead.

Reply:
@alloy yeah @orta pointed that out later IRL. I planned to make the refactor today.

Reply:
Cool, that sounds like a good plan 👍

Reply:
@alloy wondering if I'm implementing this enum correctly?

```ES6
const IntentsType = new GraphQLEnumType({
  name: "Intents",
  values: {
    BUY_ART_AND_DESIGN: {
      value: "buy art & design",
    },
    SELL_ART_AND_DESIGN: {
      value: "sell art & design",
    },
    RESEARCH_ART_PIECES: {
      value: "research art & design",
    },
    LEARN_ABOUT_ART: {
      value: "learn about art",
    },
    FIND_ART_EXHIBITS: {
      value: "find out about new exhibitions",
    },
    READ_ART_MARKET_NEWS: {
      value: "read art market news",
    },
  },
})
```

And now for the intents type I'm passing `new GraphQLList(IntentsType)`

What I'm not totally sure about is how I would actually implements this when passing `intents` as an input to the mutation...?

Reply:
Yeah this looks good, the enum definitions always look a bit awkward.

In your query you would use the capital enum name, try it in GraphiQL

Reply:
Yeah it works!